### PR TITLE
Debian packaging

### PR DESCRIPTION
**Pull request by arielshaqed**
Adds packaging for Debian-derived systems. This initial version was tested on Ubuntu (Trusty; 14.04). Will test on other Ubuntu and Debian versions.

Does not (yet) include resource limitations.

@yosih please review so we can pull

### CODE REVIEW

File: contrib/netperf/collectd-td.conf [hunk 1]
```diff
@@ -35,7 +35,7 @@ WriteQueueLimitLow 5000
 <Plugin "WriteGRPC">
   Host "cloudlatencytest-pa.googleapis.com"
   ServiceAccountJsonFile "/opt/collectd-td/etc/service-account.json"
-  RootPEMFile "/etc/ssl/certs/ca-bundle.crt"
+  RootPEMFile "@TRUSTED_ROOT_CERTS_BUNDLE@"
```
Comment (yosih) [inline contrib/netperf/collectd-td.conf:38]: Does it break the RPM package?

Comment (arielshaqed) [inline contrib/netperf/collectd-td.conf:38]: Yes. Opened #2 to handle this.

**Comment by yosih** [comment]
Tested meaning package builds + installs correctly on... GCE debian7 current image?

**Comment by arielshaqed** [comment]
Tested on Ubuntu Trusty (14.04). Ubuntu Wheezy and the Debians are next up.

**Comment by yosih** [comment]
LGTM

**Pull request merged**